schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta:
+  - installation/access_beta_distribution
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/skip_registration
+extension_module_selection:
+  - installation/module_registration/skip_module_registration
+add_on_product:
+  - installation/add_on_product_installation/accept_add_on_installation
+system_role:
```

Comment:
this is not correct. 
Could you also add the new yaml file (with keys)? You can try possible schedules using the script in `script/` folder.

Reply:
changed.